packages/wdio-browserstack-service/src/util.ts

```diff
@@ -329,7 +329,14 @@ export function getUniqueIdentifier(test: Frameworks.Test, framework?: string):
     if (framework === 'jasmine') {
         return test.fullName
     }
-    return `${test.parent} - ${test.title}`
+
+    let parentTitle = test.parent
+    // Sometimes parent will be an object instead of a string
+    if (typeof test.parent === 'object') {
+        // @ts-ignore
```

Comment:
Can we resolve the type issue?

Reply:
Actually, we are [getting the test object](https://github.com/webdriverio/webdriverio/blob/1ce750d11ad8e14486ce92a77935ebb442d11956/packages/wdio-browserstack-service/src/insights-handler.ts#L127) from the context of the test that is passed to the service (`test.ctx.test.parent`) in some places. And this object doesn't match with `Framework.Test` type which has parent has string, but in the context object, the parent can be an object. Also we don't have any exported types to type for this context.

Currently removed the ts-ignore and casted it to `any`. Let me know if you have any better solution